views/js/qtiCreator/plugins/menu/preview.js

```diff
@@ -29,10 +29,10 @@ define([
     'i18n',
```

Comment:
update the license year pls to 2016-2019

Reply:
fixed